### DISCUSSION TITLE
How to remove the black background of the footnote part?

### DISCUSSION THREAD

**Discussion by 5hEn918** [alshedivat/al-folio#2232] [Q&A] [answered]
![image](https://github.com/alshedivat/al-folio/assets/161289944/9b93122f-1bc5-4a9b-89f7-1cd651c296a8)

**Comment by george-gca** [comment, marked as answer]
The color is set in [_sass/_base.scss](https://github.com/alshedivat/al-folio/blob/70ad4ceb428bc5f2232fae240043336e5b9658b8/_sass/_base.scss#L438) at `footer.fixed-bottom`.
